transaction_test.go

```diff
@@ -60,7 +64,6 @@ func TestTransactionContext(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer tx.Rollback()
```

Comment:
Why delete this?

Reply:
Because the transaction is committed a few lines below.